src/components/AdvertiserName/BlockDropdown.tsx

```diff
@@ -34,7 +33,6 @@ const BlockDropdown = ({ id, onClickBlocked }: TBlockDropdownProps) => {
                     id={id ?? ''}
                     isBlocked={!!isBlocked}
                     isModalOpen
-                    onClickBlocked={onClickBlocked}
```

Comment:
don't we need to this to decide on the overlay?

Reply:
nopeee no more, cause i removed the useState check, now we soley check the advertiser_info response, and the mutation is handled in BlockUserModal